rewrite-java/src/main/java/org/openrewrite/java/cleanup/UseJavaInterface.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.Tree;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaVisitor;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.*;
+import org.openrewrite.marker.Markers;
+
+import java.time.Duration;
+import java.util.*;
+
+public class UseJavaInterface extends Recipe {
+
+    @Override
+    public String getDisplayName() {
+        return "Use `Deque`, `List`, `Map`, `ConcurrentMap`, `Queue`, `Set`, `SortedMap`, and `SortedSet`";
```

Comment:
This name is too long. Try for 5 words or less, per our naming guidelines. This is good information for the description.

I suggest "Use Collection Interfaces" (also suggest renaming the class itself to `UseCollectionInterfaces`)